python/src/opendp/mod.py

```diff
@@ -10,13 +10,37 @@
 import ctypes
 from typing import Any, Literal, Type, TypeVar, Union, Callable, Optional, overload, TYPE_CHECKING, cast
 
-from opendp._lib import AnyMeasurement, AnyTransformation, AnyDomain, AnyMetric, AnyMeasure, AnyFunction
+from opendp._lib import AnyMeasurement, AnyTransformation, AnyDomain, AnyMetric, AnyMeasure, AnyFunction, get_opendp_version
+
 
 # https://mypy.readthedocs.io/en/stable/runtime_troubles.html#import-cycles
 if TYPE_CHECKING:
     from opendp.typing import RuntimeType # pragma: no cover
 
 
+__all__ = [
+    'Measurement',
+    'Transformation',
+    'Queryable',
+    'Function',
+    'Domain',
+    'Metric',
+    'Measure',
+    'PrivacyProfile',
+    'PartialConstructor',
+    'UnknownTypeException',
+    'OpenDPException',
+    'GLOBAL_FEATURES',
+    'enable_features',
+    'disable_features',
+    'assert_features',
+    'binary_search_chain',
+    'binary_search_param',
+    'binary_search',
+    'exponential_bounds_search',
+    '__version__',
+]
```

Comment:
Do we still need this? It would be nice to not have to maintain it.

Reply:
It's not absolutely necessary, but I think we do need to maintain a list of exports somewhere, for a couple reasons:
- Without `__all__`, `from opendp.mod import *` brings in all of the names, including this file's own imports, which clearly don't belong in this namespace.
- `import *` only brings in names without underscores, and doesn't include `__version__`.

An alternative would be to list everything to import instead of using `*`, but splitting it between two files seems like it would be more annoying. Other possibilities:
- Don't touch `mod.py` in this PR, and instead define `__version__` in `prelude.py`.
- Don't add `__all__`, and instead add an explicit import: `from opendp.mod import __version__`.

Reply:
Great point!